services/121-service/test/helpers/program.helper.ts

```diff
@@ -517,3 +517,21 @@ export async function setAllProgramsRegistrationAttributesNonRequired(
     });
   }
 }
+
+export async function removeDeprecatedImageCodes(
+  {
+    accessToken,
+    mockCurrentDate,
+  }: { accessToken: string; mockCurrentDate?: string }, // ISO string format
```

Comment:
```suggestion
  }: {
   accessToken: string;
   mockCurrentDate?: string // ISO string format
  },
```

Wouldn't `Date` as a type work?
Or something mentioned here: https://www.webdevtutor.net/blog/typescript-type-for-date-string ?
Or this: https://gist.github.com/MrChocolatine/367fb2a35d02f6175cc8ccb3d3a20054

(Maybe not refactorings for now, but still...)

Reply:
I renamed the vairable to reflect this, and removed the comment. I left the type as string for now